client/app.js

```diff
@@ -1,26 +1,58 @@
-import React, {Component} from 'react';
+import React from 'react';
 import ReactDOM from 'react-dom';
+import ReviewList from './components/reviewList.js'
 
 class Reviews extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
-
+      currentProduct: 13,
+      currentProductReviews: [
```

Comment:
if you want to  clean up some of your code you can create a variable outside of your class component with your array of two objects. Then you can set currentProductReviews  equal to that variable. Just a trick Arohan showed me that I found valuable.

Reply:
Sweet, that's a good idea.